runtime/gc_modron_startup/mminit.cpp

```diff
@@ -2927,7 +2927,7 @@ gcInitializeDefaults(J9JavaVM* vm)
 
 				if (hwSupported) {
 					/* Software Barrier request overwrites HW usage on supported HW */
-					extensions->concurrentScavengerHWSupport = hwSupported && !extensions->softwareRangeCheckReadBarrier;
+					extensions->concurrentScavengerHWSupport = hwSupported && !extensions->softwareRangeCheckReadBarrier && !J9_ARE_ANY_BITS_SET(vm->extendedRuntimeFlags2, J9_EXTENDED_RUNTIME2_ENABLE_AOT);
```

Comment:
Does this code change the behavior when AOT is enabled? We want a change in behavior if AOT **and** portableAOT enabled.

Reply:
Right. Is it the same flag we check to force 3-bit shift?

Reply:
You are checking `J9_EXTENDED_RUNTIME2_ENABLE_PORTABLE_SHARED_CACHE` to force shift selection. Should check here be the same?

Reply:
Yes sorry I used the wrong flag, should be `J9_EXTENDED_RUNTIME2_ENABLE_PORTABLE_SHARED_CACHE`